neptune/alpha/internal/operation.py

```diff
@@ -133,18 +133,20 @@ def from_dict(data: dict) -> 'UploadFile':
 class UploadFileSet(Operation):
 
     file_globs: Sequence[str]
+    reset: bool
```

Comment:
If you add reset to UploadFileSet, you should handle these parameter in backend mock to add file_globs instead replacing.